man/systemd-system.conf.xml

```diff
@@ -171,6 +171,126 @@
         effect if a hardware watchdog is not available.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>IOCostQOSPath=</varname></term>
+        <listitem><para> Configure the absolute path to device to be configured
+        in io.cost.qos file of the root cgroup.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostQOSCtrl=</varname></term>
+        <listitem><para>Configure whether the io.cost.qos parameters are
+        controlled by the kernel or by the user. Allowed values are 'auto' and 'user'.
+        <varname>IOCostQOSRPct</varname>, <varname>IOCostQOSRLat</varname>, <varname>IOCostQOSWPct</varname>,
+        <varname>IOCostQOSRLat</varname>, <varname>IOCostQOSWPct</varname>, <varname>IOCostQOSWLat</varname>
+        <varname>IOCostQOSMin</varname>, and <varname>IOCostQOSMax</varname> are all ignored if value is set to 'auto'. </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostQOSEnabled=</varname></term>
+        <listitem><para>Configure whether the io.cost.qos parameters are enabled.
+        Values are 'yes' and 'no' and set to 'no' by default. All other IOCostQOS fields
+        are ignored if this is set to 'no'.
```

Comment:
I am confused. What is IOCostQOSCtrl=, and what is IOCostQOSEnabled=? Superficially they sound like doing the exact same thing?

i.e. the docs should be good enough that I grok what this does, at the bare minimum. Right now I have no idea, this raises more questions than it answers.